content/applications/inventory_and_mrp/quality/quality_check_types/instructions_check.rst

```diff
@@ -0,0 +1,86 @@
+==========================
+Instructions quality check
+==========================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+
+In Odoo *Quality*, an *Instructions* check is one of the quality check types that can be selected
+when creating a new quality check or Quality Control Point (QCP). Instructions checks consist of a
+text entry field that allows the creator to provide instructions for how to complete the check.
+
+For a full overview of how to configure a quality check or a |QCP|, see the documentation on
+:ref:`Quality checks <quality/quality_management/quality-checks>` and :ref:`Quality Control
+Points <quality/quality_management/quality-control-points>`.
+
+Process an Instructions quality check
+=====================================
+
+There are multiple ways that Instructions quality checks can be processed. If a quality check is
+assigned to a specific manufacturing, inventory, or work order, the check can be processed on the
+order itself. Alternatively, a check can be processed from the check's page.
+
+Process from the quality check's page
+-------------------------------------
+
+To process an Instructions quality check from the check's page, begin by navigating to
+:menuselection:`Quality --> Quality Control --> Quality Checks`, and select a quality check. Follow
+the :guilabel:`Instructions` for how to complete the check.
+
+If the product passes the check, click the :guilabel:`Pass` button above the quality check form. If
+the product does not pass the check, click the :guilabel:`Fail` button, instead.
+
+Process a quality check on an order
+-----------------------------------
+
+To process an Instructions quality check on an order, select a manufacturing order or inventory
+order (receipt, delivery, return, etc.) for which a check is required. Manufacturing orders can be
+selected by navigating to :menuselection:`Manufacturing --> Operations --> Manufacturing Orders`,
+and clicking on an order. Inventory orders can be selected by navigating to
+:menuselection:`Inventory`, clicking the :guilabel:`# To Process` button on an operation card, and
+selecting an order.
+
+On the selected manufacturing or inventory order, a purple :guilabel:`Quality Checks` button appears
+above the order. Click the button to open the :guilabel:`Quality Check` pop-up window, from which
+any quality checks created for the order can be processed.
+
+.. image:: instructions_check/quality-check-pop-up.png
+   :align: center
+   :alt: The Quality Check pop-up window on a manufacturing or inventory order.
+
+To complete an Instructions quality check, follow the instructions detailed in the
+:guilabel:`Quality Check` pop-up window. Finally, click :guilabel:`Validate` to confirm that the
+check has been completed.
+
+If a quality alert must be created, click the :guilabel:`Quality Alert` button that appears at the
```

Comment:
As someone not super familiar with Quality, it's not obvious why a quality alert is raised at this step. Maybe starting this paragraph with a little bit more context on when Quality Alerts are raised, it would be helpful (::

```suggestion
If the quality check failed, raise a quality alert to ____. To do so, click the :guilabel:`Quality Alert` button that appears at the...
```
(I left the explanation empty because i dont know :joy: )